xAPI-Data.md

```diff
@@ -313,21 +313,21 @@ The details of each property of a statement are described in the table below.
 <table>
 	<tr><th>Property</th><th>Type</th><th>Description</th><th>Required</th></tr>
 	<tr><td>id</td><td>UUID</td>
-	<td>UUID assigned by LRS if not set by the Activity Provider.</td>
+	<td>UUID assigned by LRS if not set by the Learning Record Provider.</td>
 	<td>Recommended</td></tr>
 	<tr><td><a href="#actor">actor</a></td><td>Object</td>
 	<td>Whom the Statement is about, as an <a href="#agent">Agent</a> or 
-		<a href="#group">Group</a> Object. Represents the "I" in "I Did This".</td>
+		<a href="#group">Group</a> Object.</td>
```

Comment:
Looking at this again in more detail, I'm ok with removing it from the table. We still have the analogy in the definitions section, which i think is a good place for it. 

We also have I Did This in the description for object, but not for actor or verb, which is a little weird. Maybe we should remove it from object too. We should replace it with "The object defines the thing that was acted on" in line with the actor and verb sections.